Jenkinsfile

```diff
@@ -277,11 +325,29 @@ pipeline {
                                 }
                             }
                         }
+                        stage("archive-release") {
```

Comment:
`archive` can be one single step as discussed going into `${OUT_DIR}/*.dmg` and `${OUT_DIR}/*.pkg`